src/main/java/org/jenkins/tools/test/logging/LoggingConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package org.jenkins.tools.test.logging;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.util.MissingResourceException;
+import java.util.logging.LogManager;
+
+/**
+ * A logging configuration class suitable for passing to {@code -Djava.util.logging.config.class}.
+ *
+ * @see LogManager#readConfiguration()
+ */
+public class LoggingConfiguration {
```

Comment:
Our new custom logging configuration class. This just loads `logging.properties` from the classpath and then applies it with `LogManager#updateConfiguration`. Preferring `#updateConfiguration` to `#readConfiguration` for the reasons described [here](https://stackoverflow.com/a/36069135).